frontend/src/pages/GraphPF/GraphPF.tsx

```diff
@@ -322,11 +322,13 @@ const TopologyContent: React.FC<{
     }
 
     if (layoutInProgress !== LayoutType.LayoutNoFit) {
+      controller.getGraph().fit(FIT_PADDING);
```

Comment:
This needs some refactoring. If the layout type is not Resize, you will fit the layout twice without any delay.

My suggestion (simply remove the else statement in the condition):

```
if (layoutInProgress !== LayoutType.LayoutNoFit) {
      controller.getGraph().fit(FIT_PADDING);

      // On a resize, delay fit to ensure that the canvas size updates before the fit
      if (layoutInProgress === LayoutType.Resize) {
        setTimeout(() => {
          controller.getGraph().fit(FIT_PADDING);
        }, 500);
      }
    }
```

Reply:
@ferhoyos OK, I cleaned it up.  I made one additional small tweak to the padding size, but I don't see anything else to do at this time.